### PR TITLE
Added popup.html

### LINKED ISSUE #20: Add a popup.html
Add a popup.html on click, which defines "How to?" and further instructions and Repo link etc.

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes #20 

screenshot 12

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Instagram Downloader</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+        crossorigin="anonymous">
+</head>
+
+<body>
+    <h1 id="title">Instagram Downloader.</h1>
```
Comment (aashutoshrathi) [inline popup.html]: Ahh.. remove full stop

File: popup.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <p>Download: Downloads the picture</p>
+    </section>
+    <hr>
+    <footer>
+        <a href="https://github.com/aashutoshrathi/Insta-Downloader-Extension" target="_blank"><i class="fab fa-github fa-3x"
+                style="color: white"></i></a>
+    </footer>
+</body>
+
+</html>
```
Comment (aashutoshrathi) [inline popup.html]: Add newline

File: style.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.info{
+    font-size: 20px;
+}
+
+footer{
+    margin-top: 70px;
+    margin-left: 100px;
+}
```
Comment (aashutoshrathi) [inline style.css]: newline here too.

File: popup.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <title>Instagram Downloader</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+        crossorigin="anonymous">
+</head>
+
+<body>
+    <h1 id="title">Instagram Downloader.</h1>
+    <hr>
+    <section class="info">
```
Comment (aashutoshrathi) [inline popup.html]: Decrease the font size for section.

**Comment by nityanandagohain** [comment]
@aashutoshrathi I have made the changes please review.

**Comment by aashutoshrathi** [comment]
@nityanandagohain Can you provide a GIF?

**Comment by nityanandagohain** [comment]
first

😄

**Comment by aashutoshrathi** [comment]
What happened @nityanandagohain

**Pull request merged**